swan-lake/resources/featured-scenarios/build-a-data-service-in-ballerina.md

```diff
@@ -59,224 +55,229 @@ CREATE TABLE Company.Employees (
 
 ## Create the service package
 
-Ballerina uses packages to group code. You need to create a Ballerina package and write the business logic in it. In the terminal, execute the command below to create the Ballerina package for the implementation.
+Ballerina uses packages to group code. You need to create a Ballerina package and write the business logic in it. 
 
-> **Info:** For more information on Ballerina packages, see [Organize Ballerina code](/learn/organize-ballerina-code/).
+1. In the Terminal, execute the command below to create the Ballerina package for the implementation.
 
-```
-$ bal new data_service
-``` 
+    > **Info:** For more information on Ballerina packages, see [Organize Ballerina code](/learn/organize-ballerina-code/).
 
-This creates a directory named `data_service` with the files below.
+    ```
+    $ bal new build-a-data-service
```

Comment:
Why the package is named differently? We usually don't use verbs in naming

Reply:
Updated.